server/src/weaverbird/pipeline/steps/ifthenelse.py

```diff
@@ -22,13 +22,13 @@ class Config(PopulatedWithFieldnames):
 
 
 class IfthenelseStep(BaseStep, IfThenElse):
-    class Config(PopulatedWithFieldnames):
-        ...
-
     name: Literal["ifthenelse"] = "ifthenelse"
     new_column: ColumnName = Field(alias="newColumn")
 
+    class Config(BaseConfig):
+        allow_population_by_field_name = True
+
 
 class IfThenElseStepWithVariables(IfthenelseStep, StepWithVariablesMixin):
-    class Config(PopulatedWithFieldnames):
-        ...
+    class Config(BaseConfig):
+        allow_population_by_field_name = True
```

Comment:
Same remark here, It's already in the parent

Reply:
Same remark here as well :P